components/button.css

```diff
@@ -4,6 +4,13 @@
   border-radius: 5px;
   cursor: pointer;
   font-weight: 600;
+  background: #3b3b3b;
```

Comment:
Suggestion: Use variables instead of hard code string.(Maybe you forgot to do it several places.)

Reply:
yes will do that, at this point i am running out of variable names